astropy/visualization/wcsaxes/formatter_locator.py

```diff
@@ -57,6 +58,13 @@
 }
 
 
+def _fix_minus(labels: list[str], /) -> list[str]:
+    # correctly support axes.unicode_minus, but do it in a
+    # way that preserves arbitrary separators: only fix the leading character
+    # see https://github.com/astropy/astropy/issues/15898
+    return [Formatter.fix_minus(s[0]) + s[1:] for s in labels]
```

Comment:
In this case, maybe it is better to just implement the logic here?
```suggestion
    if not rcParams['axes.unicode_minus']:
        return labels
    minus = "\N{MINUS SIGN}"
    return [minus + s[1:] if s.startswith("-") else s for s in labels]
```

Reply:
I assume `matplotlib.ticker.Formatter.fix_minus` checks the `rcParams`?

Reply:
It does ! see https://github.com/matplotlib/matplotlib/blob/fd5f1640cc1f3b5676d3242e10bd069e18d75f6c/lib/matplotlib/ticker.py#L246-L260

Reply:
Thanks!